### ISSUE TITLE
🐛 [Bug] Shortened text is shown longer than full text

### ISSUE THREAD

**Issue by Tirokk** [Human-Connection-Community/Human-Connection-New-20201003#1795]
> [<img alt="datenbrei" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/datenbrei) **Authored by [datenbrei](https://github.com/datenbrei)**
_<time datetime="2019-10-02T13:52:30Z" title="Wednesday, October 2nd 2019, 3:52:30 pm +02:00">Oct 2, 2019</time>_
_Closed <time datetime="2019-10-17T10:21:44Z" title="Thursday, October 17th 2019, 12:21:44 pm +02:00">Oct 17, 2019</time>_
---

## :bug: Bugreport
1. If a text needs no shortening, it should not have an option to be shown full.
2. If 

### Steps to reproduce the behavior
1. Write a comment, which is three lines long
2. Save
3. View: It can be shown full although it is already fully visible.
Normally shortened version:
![short](https://user-images.githubusercontent.com/13457963/66049802-5cd07d80-e52c-11e9-8e76-a4c53290f589.png)

Normally full version:
![long](https://user-images.githubusercontent.com/13457963/66049824-65c14f00-e52c-11e9-92cc-8bcf3a7ad855.png)

### Expected behavior
The short version should not be longer than the full version

### Version & Environment
 Type: [Desktop]      
 - OS: [Linux]      
 - Browser: [Firefox] 
 - Version [69]  
 - Device: [Notebook]  

### Additional context
It may have to do with the shortened version showing dots, which opnes a new line.

**Comment by Tirokk** [comment]
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-02T21:51:33Z" title="Wednesday, October 2nd 2019, 11:51:33 pm +02:00">Oct 2, 2019</time>_

---


I guess I don't understand what you mean by this @datenbrei 
At the moment we are truncating the comment's content at 180 characters, which can be increased, if you think it's too few? Or do you mean something else?

**Comment by Tirokk** [comment]
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-10-09T07:14:27Z" title="Wednesday, October 9th 2019, 9:14:27 am +02:00">Oct 9, 2019</time>_

---


it looks that way to me
![FireShot Capture 066 - 123456789012345678901234567890123456789012345678901234567890123456789_ - localhost](https://user-images.githubusercontent.com/1324583/66459353-29827700-ea75-11e9-8ad0-6c0871c697da.png)

**Comment by Tirokk** [comment]
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-10-09T10:04:14Z" title="Wednesday, October 9th 2019, 12:04:14 pm +02:00">Oct 9, 2019</time>_

---


ok verstehe, the three points for truncated text are there twice.

**Comment by Tirokk** [comment]
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-10-09T12:59:22Z" title="Wednesday, October 9th 2019, 2:59:22 pm +02:00">Oct 9, 2019</time>_

---


We've got another whole other problem.
we have to look how many lines a comment has. currently there can be a character in every line and the comment can be up to 180 lines long. 
i would suggest to block any formatting in the comments. remove line breaks and empty lines. 
simply pure text 

![FireShot Capture 068 - This is post #7 - Human Connection - localhost](https://user-images.githubusercontent.com/1324583/66483330-1f786c80-eaa5-11e9-960d-4c79c1e86ee7.png)

**Comment by Tirokk** [comment]
> [<img alt="datenbrei" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/datenbrei) **Authored by [datenbrei](https://github.com/datenbrei)**
_<time datetime="2019-10-09T13:38:09Z" title="Wednesday, October 9th 2019, 3:38:09 pm +02:00">Oct 9, 2019</time>_

---


Why? What's bad with formatted comments? And line breaks are a must have.
